tests/utils/date-service.test.ts

```diff
@@ -0,0 +1,44 @@
+import { getTimeTrackedInSeconds } from '@utils/date-service';
+
+describe('date-service tests', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getIssueWorkedSeconds method', () => {
+    it('should return a positive value if no end time provided', async (): Promise<void> => {
+      // given
+      const startTime: Date = new Date('2024-03-12T09:00:00Z');
+      const zeroSeconds: number = 0;
```

Comment:
Why would you need a variable for the number zero? No need for this.

Reply:
done